### DISCUSSION TITLE
能否支持自定义二级属性的分割符？

### DISCUSSION THREAD

**Discussion by daiwenzh5** [liuzhihang/doc-view#45] [Ideas]
当前使用的 `-->`，在 confluence 不太好看。
![Snipaste_2022-03-31_16-34-49](https://user-images.githubusercontent.com/29970117/161013106-f6af8963-397b-430c-b373-d04605dafdeb.png)
感觉还是这个 `└` 好一点

**Comment by liuzhihang** [comment]
收到
